cdip_connector/core/connector_base.py

```diff
@@ -50,12 +40,11 @@ async def main(self) -> None:
                 logger.info(f'CLIENT_ID: {cdip_settings.KEYCLOAK_CLIENT_ID}')
                 integration_info = await self.portal.get_authorized_integrations(session)
 
-                # todo gather_with_semaphore!
-                result = await asyncio.gather(*[
-                    self.extract_load(session, i) for i in integration_info
-                ])
-
+                result = [await self.extract_load(session, i) for i in integration_info]
+                # tasks = [asyncio.ensure_future(self.extract_load(session, i)) for i in integration_info]
+                # result = await asyncio.gather(*tasks)
```

Comment:
This gather was fraught since it was unbounded. Running for several integrations at once would overload our portal server. So I've serialized it here until we can refactor it to use a Semaphore properly.